waiter/config-full.edn

```diff
@@ -350,6 +350,11 @@
                                 "scale-factor" 1
                                 "scale-up-factor" 0.1}
 
+ ;; Service description parameters with numeric values, e.g. cpus, can be configured to have
+ ;; maximum value limits. When not configured, these defaults will be used:
```

Comment:
> When not configured, these defaults will be used

This sentence is confusing. It makes it sound like this section is for the defaults, but some other section is for configuring the maximums. I think we could just remove that sentence altogether, no?

Reply:
Removed.